stable/fairwinds-insights/README.md

```diff
@@ -126,7 +126,9 @@ See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/technical-
 | ingress.separate | bool | `false` | Create different Ingress objects for the API and dashboard - this allows them to have different annotations |
 | ingress.extraPaths | object | `{}` | Adds additional path ie. Redirect path for ALB |
 | postgresql.postMigrate | bool | `false` | Set to `true` to run migrations after the upgrade |
-| postgresql.image.tag | string | `"14.2.0-debian-10-r94"` |  |
+| postgresql.image.registry | string | `"quay.io"` |  |
+| postgresql.image.repository | string | `"fairwinds/postgres-partman"` |  |
```

Comment:
As we might, in the future, add more extensions to this image (`pg_cron`)... I wonder if we could use `postgres-libraries` or some more generic naming for it

Reply:
Right, we can refactor it, when we add pg_cron